website/docs/d/route53_record.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: "Route 53"
+layout: "aws"
+page_title: "AWS: aws_route53_record"
+description: |-
+  Terraform data source for managing an AWS Route 53 Record.
```

Comment:
not sure about the managing
```suggestion
  Terraform data source for an AWS Route 53 Record.
```
see examples of similar
https://github.com/hashicorp/terraform-provider-aws/blob/c8bf2a3901c413ce5974efa2bd2733ef3461203b/website/docs/d/s3_objects.html.markdown#data-source-aws_s3_objects
https://github.com/hashicorp/terraform-provider-aws/blob/c8bf2a3901c413ce5974efa2bd2733ef3461203b/website/docs/d/route_tables.html.markdown#data-source-aws_route_tables